vnext/Desktop/Modules/WebSocketModule.cpp

```diff
@@ -24,7 +25,15 @@ constexpr char moduleName[] = "WebSocketModule";
 
 namespace Microsoft::React {
 
-WebSocketModule::WebSocketModule() {}
+WebSocketModule::WebSocketModule()
+    : m_resourceFactory{[](const string &url, bool legacyImplementation, bool acceptSelfSigned) {
+        return IWebSocketResource::Make(url, legacyImplementation, acceptSelfSigned);
+      }} {}
```

Comment:
(nit) parameter independent settings like this should be specified in .h file.
For constructor initialization, it's better to only specify data that comes with constructor's arguments